app/controllers/browse_controller.rb

```diff
@@ -34,8 +34,8 @@ def show
   def show_html(page)
     taxon_resolver = TaxonRedirectResolver.new(
       request,
-      is_page_in_ab_test: lambda { params[:top_level_slug] == "education" },
-      map_to_taxon: lambda { "education" }
+      is_page_in_ab_test: lambda { !redirects[params[:top_level_slug]].nil? },
```

Comment:
the `!(expression).nil?` construct is equivalent in most contexts to `expression`